dnsaas/api/v2/serializers.py

```diff
@@ -115,6 +126,27 @@ def _clean_txt_content(self, record_type, attrs):
         if record_type == 'TXT':
             attrs['content'] = attrs['content'].replace('\\', '')
 
+    def _validate_service(self, attrs):
+        if (
+            # enforce service required for patch updates
```

Comment:
why? shouldn't it be required for PUT requests only? what request are we sending from GUI?